coda/templates/mdstore/bag_search_results.html

```diff
@@ -50,6 +53,9 @@
                     <td class="data" style="vertical-align:middle">
                         <i class="icon-globe"></i> <a href="http://{{ request.META.HTTP_HOST }}/APP/bag/{{ entry.name }}">ATOM</a>
                     </td>
+                    <td class="data" style="vertical-align:middle">
+                        <i class="icon-globe"></i> <a href="http://{{ request.META.HTTP_HOST }}/bag/{{ entry.name }}/links/">links</a>

```

Comment:
What do you think about changing the links for this view and the others to use the `{% url %}` tag?